specs.md

```diff
@@ -53,15 +53,15 @@ FSM-0 | Startup CAN message | The FSM must transmit a startup message over CAN o
 FSM-5 | APPS open/short circuit | If there is an open/short circuit in either encoder the FSM must report 0% mapped pedal percentage. | T.4.2.2, T.4.2.9
 FSM-6 | APPS disagreement | - When the primary and secondary APPS positions disagree by more then 10%, and the disagreement persists for more than 100ms, the FSM must throw a motor shutdown fault and report 0% mapped pedal percentage. <br/> - The FSM must clear the motor shutdown fault after the APPS positions agree within 10%, and the agreement persists for more than 1s. | T.4.2.3, T.4.2.4, T.4.2.5
 FSM-7 | APPS/brake pedal plausibility check | - When the APPS senses brake actuation and more than 25% mapped pedal percentage simultaneously, the FSM must throw a motor shutdown fault and report 0% mapped pedal percentage. <br/> - The FSM must clear the motor shutdown fault after the APPS senses less than 5% mapped pedal percentage, regardless of the brake state. | EV.3.4.1, EV.3.4.2
-FSM-8 | Steering angle reporting | - The FSM must report the steering angle in degrees over CAN at 1kHz or faster, where 0 degrees represents straight wheels and a clockwise turn of the steering wheel corresponds to an increase in steering angle. <br/> - The FSM must send a non-critical fault when the steering angle is beyond the max turning radius of the steering wheel.
-FSM-9 | Wheel speed reporting | - The FSM must report the two front wheel speeds in km/h over CAN at 1kHz or faster. <br/> - The FSM must send a non-critical fault when either front wheel speed is below -10km/h or above 150km/h.
+FSM-8 | Steering angle reporting | - The FSM must report the steering angle in degrees over CAN at 100Hz or faster, where 0 degrees represents straight wheels and a clockwise turn of the steering wheel corresponds to an increase in steering angle. <br/> - The FSM must send a non-critical fault when the steering angle is beyond the max turning radius of the steering wheel.
+FSM-9 | Wheel speed reporting | - The FSM must report the two front wheel speeds in km/h over CAN at 100Hz or faster. <br/> - The FSM must send a non-critical fault when either front wheel speed is above 150km/h.
 FSM-16 | Open circuit encoder alarm | - When the primary or secondary APPS encoder alarm goes high for 10ms continuously, throw a motor shutdown fault and report 0% mapped pedal percentage. <br/> - When both the primary and secondary APPS encoder alarms go low for 10ms continuously, clear the motor shutdown fault. | T.4.2.9
-FSM-17 | Brake actuation reporting | - The FSM must report the brake actuation ON/OFF status over CAN at 100Hz or faster
+FSM-18 | Brake actuation reporting | - The FSM must report the brake actuation ON/OFF status over CAN at 100Hz or faster
```

Comment:
Unrelated to this PR but CAN ID is the same as `Entering the AIR-open state`. Changed to FSM-18 as the latter ID was added first